app/components/Sidebar/components/Collections.js

```diff
@@ -132,19 +126,16 @@ class CollectionLink extends Component {
         >
           <CollectionName justify="space-between">
             {collection.name}
-
-            <CollectionAction>
-              <CollectionMenu
-                history={history}
-                collection={collection}
-                onOpen={() => (this.menuOpen = true)}
-                onClose={() => (this.menuOpen = false)}
-                onImport={this.handleImport}
-                open={this.menuOpen}
-              />
-            </CollectionAction>
           </CollectionName>
         </SidebarLink>
+        <CollectionAction>
```

Comment:
Had to move this out as the file input doesn't work inside the `SidebarLink` anchor, not a big deal though.